websubhub-ballerina/build.gradle

```diff
@@ -127,19 +151,48 @@ task revertTomlFile {
     }
 }
 
+task ballerinaTest {
```

Comment:
This task should depends on `:websubhub-native:build` task. Let's add it here or in line #274 as `ballerinaTest.dependsOn ":websubhub-native:build"`

Reply:
* Added the required configuration to `gradle.build`.